pkg/internal/resources/api-server/apiserver.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2019 The KubeCarrier Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apiserver
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"sigs.k8s.io/kustomize/v3/pkg/image"
+	"sigs.k8s.io/kustomize/v3/pkg/types"
+
+	"github.com/kubermatic/kubecarrier/pkg/internal/kustomize"
+	"github.com/kubermatic/kubecarrier/pkg/internal/resources/constants"
+	"github.com/kubermatic/kubecarrier/pkg/internal/version"
+)
+
+// Config holds the config information to generate the KubeCarrier master controller manager setup.
+type Config struct {
+	// Namespace is the KubeCarrier master controller manager should be deployed into.
+	Namespace string
+	// Name of this Tower object.
```

Comment:
Please pay more attention to your PR... and remove all the occurrences of `tower`.

Reply:
I did...but forgot the resources subpackage.